imports/plugins/custom/authentication/client/component/authentication/signup.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import { Components, replaceComponent, getRawComponent } from "@reactioncommerce/reaction-components";
+
+const SignUp = getRawComponent("SignUp");
+
+class MySignUp extends SignUp {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      seller: false
+    };
+  }
+
+  handleSubmit = (event) => {
+    if (this.props.onFormSubmit) {
+      this.props.onFormSubmit(event, this.state.email, this.state.password, this.state.seller);
+    }
+  }
+
+  renderFormMessages() {
```

Comment:
This method is not different from the base class implementation. Is there any need to duplicate it here